torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -36,6 +36,16 @@ RegisterOperators reg(
            };
          },
          aliasAnalysisSpecialCase()),
+     Operator(
+         prim::profile_ivalue,
+         [](const Node* node) -> Operation {
+           auto callback = node->cast<ProfileIValueOp>()->getCallback();
```

Comment:
Good catch. It probably should be removed now since profile is lowered to interpreter's instruction. Same with prim::profile & prim::profile_optional :stuck_out_tongue: